src/send_data.py

```diff
@@ -1,71 +1,43 @@
-from sensor_temp import getTemp
-from sensor_motion import getMotion
-from alarm import alarm_on
-from button import button_pressed
 import requests
-from pprint import pprint
 from requests import RequestException
-import time
-import datetime
-import os
-from dotenv import load_dotenv
-
-load_dotenv()
-
-URL = os.getenv("SERVER_URL")
-DEVICE_ID = '1'
+from pprint import pprint
+from time import sleep
+# import os
+# from dotenv import load_dotenv
 
-## SEND DATA OUT
-def create_sensor_data_dict():
-    """Create data from multiple sensors
+# load_dotenv()
 
-    Raises:
-        e: Failed to get data from sensor
+# URL = os.getenv("SERVER_URL")
+# DEVICE_ID = '1'
 
-    Returns:
-        dict: Dictionary of sensor data
-    """
-    
+def send_data(data, URL, DEVICE_ID):
     try:
```

Comment:
In this scope, `URL` and `DEVICE_ID` is not a global variable.
Make it lowercase.

Reply:
Done